src/Services/Account/Command/Application/Services/ApplicationService.cs

```diff
@@ -31,4 +34,23 @@ public Task AppendEventsAsync(IAggregateRoot aggregate, CancellationToken cancel
                 await _eventBusGateway.PublishAsync(aggregate.Events.Select(tuple => tuple.@event), ct);
             },
             cancellationToken: cancellationToken);
+
+    public async Task StreamReplayAsync(string name, Guid? id, CancellationToken cancellationToken)
+    {
+        if (id.HasValue)
+            await _eventBusGateway.PublishAsync(new NotificationEvent.RebuildProjectionRequested(id.Value, name), cancellationToken);
+        else
+            await foreach (var accountId in _eventStoreGateway.GetAggregateIdsAsync(cancellationToken))
+                await _eventBusGateway.PublishAsync(new NotificationEvent.RebuildProjectionRequested(accountId, name), cancellationToken);
+    }
```

Comment:
I'm not sure if rebuilding just a specific Aggregate projection makes any sense. In my view, if the projection has changed, every instance must change.